src/CliCore.ts

```diff
@@ -114,11 +114,11 @@ export default class CliCore {
 
         if (oneshot === true) {
```

Comment:
It looks odd to me that we have this repeated block of code. In oneshot mode we don't catch an error (I guess it crashes on the command line?) and if its not oneshot mode then we return an error. This seems weird. Can we come up with a way where there is just one logic? Either the method catches error and returns it, or it allows the error to bubble up.

Reply:
removed duplicated code